bigquery_etl/glam/templates/extract_sample_counts_v1.sql

```diff
@@ -20,4 +20,14 @@ AND total_sample IS NOT NULL
 WHERE
   cp.app_version IS NOT NULL
   AND total_users > {{ total_users }}
-  AND cp.client_agg_type not in ('sum','min','avg','max')
\ No newline at end of file
+  AND cp.client_agg_type not in ('sum','min','avg','max')
+GROUP BY
+  cp.channel,
+  cp.ping_type,
+  cp.app_version,
+  cp.app_build_id,
+  cp.os,
+  cp.metric,
+  cp.key,
+  client_agg_type,
+  total_sample
```

Comment:
Is the grouping here used to filter out duplicate entries?

Reply:
yes you are correct